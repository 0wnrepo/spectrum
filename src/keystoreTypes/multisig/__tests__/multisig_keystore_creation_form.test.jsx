jest.mock('../../../helpers/web3/connect', () => require('../../../../test/__mocks__/connectMock'));

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import MultisigKeystoreCreationForm from '../multisig_keystore_creation_form';

describe('<MultisigKeystoreCreationForm />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <MultisigKeystoreCreationForm
        formData={{}}
        formChange={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
