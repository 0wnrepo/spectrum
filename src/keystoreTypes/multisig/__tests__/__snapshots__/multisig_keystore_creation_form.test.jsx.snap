// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultisigKeystoreCreationForm /> renders correctly 1`] = `
<Grid
  stackable={true}
  textAlign="center"
>
  <GridColumn
    width={8}
  >
    <Header
      as="h3"
      icon={true}
    >
      <Icon
        as="i"
        color="blue"
        name="upload"
      />
      Deploy New Contract
      <HeaderSubheader
        content="Create a new contract and configure owners"
      />
    </Header>
    <Connect(NetworkSelector)
      onChange={[Function]}
    />
  </GridColumn>
  <GridColumn
    width={8}
  >
    <Header
      as="h3"
      icon={true}
    >
      <Icon
        as="i"
        color="orange"
        name="linkify"
      />
      Use Existing Address
      <HeaderSubheader
        content="Add an already-deployed multisig wallet instance"
      />
    </Header>
    <AddressInput
      formChange={[Function]}
      formData={Object {}}
      hideDropdown={true}
      name="address"
      placeholder="Enter Conrtract Address"
      showQrScanner={true}
    />
  </GridColumn>
</Grid>
`;
